File: smelly/assertion-roulette/java-junit/TestEventHandlerService_testService.java
public class TestEventHandlerService extends XDataTestCase {
    @Test
    public void testService() throws Exception {
        EventHandlerService ehs = _testEventHandlerService();
        // check default initializations
        assertTrue(ehs.getEventQueue() instanceof MemoryEventQueue);
        Set<String> jobtypes = ehs.getAppTypes();
        assertTrue(jobtypes.contains("workflow_job"));
        assertTrue(jobtypes.contains("coordinator_action"));

        Services services = Services.get();
        services.destroy();
        services = new Services();
        Configuration conf = services.getConf();
        conf.set(Services.CONF_SERVICE_EXT_CLASSES, "");
        services.init();
        assertFalse(EventHandlerService.isEnabled());
    }
}

File: assertion-roulette/CSharpXUnit/CSharpXUnit.Tests/Tests/TestEventHandlerService_testService.cs
public class TestEventHandlerService : XDataTestCase
{
    [Fact]
    public void TestService()
    {
        EventHandlerService ehs = _testEventHandlerService();
        // check default initializations
        Assert.IsType<MemoryEventQueue>(ehs.GetEventQueue());
        var jobtypes = ehs.GetAppTypes();
        Assert.Contains("workflow_job", jobtypes);
        Assert.Contains("coordinator_action", jobtypes);

        Services services = Services.Get();
        services.Destroy();
        services = new Services();
        Configuration conf = services.GetConf();
        conf.Set(Services.CONF_SERVICE_EXT_CLASSES, "");
        services.Init();
        Assert.False(EventHandlerService.IsEnabled());
    }
}
